infrastructures/infrastructure-factory/src/main/java/org/eclipse/che/api/factory/server/scm/kubernetes/KubernetesGitCredentialManager.java

```diff
@@ -124,11 +130,16 @@ public void createOrReplace(PersonalAccessToken personalAccessToken)
                 annotations.put(ANNOTATION_SCM_URL, personalAccessToken.getScmProviderUrl());
                 annotations.put(ANNOTATION_SCM_USERNAME, personalAccessToken.getScmUserName());
                 annotations.put(ANNOTATION_CHE_USERID, personalAccessToken.getCheUserId());
+                annotations.put(ANNOTATION_DEV_WORKSPACE_MOUNT_PATH, CREDENTIALS_MOUNT_PATH);
+                // Adding devworkspace label here and not in the default map,
+                // in case of a secret from previous version that does not have it
+                Map<String, String> labels = new HashMap<>(LABELS);
+                labels.put(LABEL_DEV_WORKSPACE_CREDENTIAL, "true");
```

Comment:
It's not clear to me, why we can't use the original `LABELS` map here. The `(LABEL_DEV_WORKSPACE_CREDENTIAL, "true")` looks like a constant, so why we can't add it in the `LABELS` directly?

Reply:
Because we use `LABELS` here  https://github.com/eclipse-che/che-server/blob/f2bceb8b265f671a716040e7891b71eb005f1c9b/infrastructures/infrastructure-factory/src/main/java/org/eclipse/che/api/factory/server/scm/kubernetes/KubernetesGitCredentialManager.java#L96
while searching for already existing git credential secret.
There might be this secret but does not have the DW label because it was created before this PR.